source/expressions.tex

```diff
@@ -3408,13 +3408,13 @@
 to type \tcode{\keyword{void}*}\iref{conv.ptr}.
 After these conversions, if the
 argument does not have arithmetic, enumeration, pointer, pointer-to-member,
-or class type, the program is ill-formed. Passing a potentially-evaluated
-argument
-of a scoped enumeration type or
-of a class type\iref{class} having an eligible non-trivial
-copy constructor, an eligible non-trivial move constructor,
-or a
-non-trivial destructor\iref{special},
+or class type, the program is ill-formed.
+Passing a potentially-evaluated argument
+of a scoped enumeration type\iref{dcl.enum} or
+of a class type\iref{class} having
+an eligible\iref{special} non-trivial copy constructor\iref{class.copy.ctor},
```

Comment:
I don't like splitting adjective + noun with a cross-reference if we can help it.
I think it's good enough if we use \iref{special,class.copy.ctor} after the phrase.